operator/src/test/java/org/bf2/operator/operands/KafkaClusterTest.java

```diff
@@ -341,6 +348,11 @@ void testDeveloperManagedKafka() throws IOException {
     @Test
     void testManagedKafkaToKafkaWithCustomConfiguration() throws IOException {
         ManagedKafka mk = exampleManagedKafka("60Gi");
+        Map<String, Object> brokerConfig = new HashMap<>();
+        brokerConfig.put("broker.foo", "bar");
+        brokerConfig.put("auto.create.topics.enable", null);
+        configureMockOverrideManager(mk, brokerConfig);
```

Comment:
ooft. The assertion of that is a bit subtle for my taste. 

Given we explicitly set properties here I'd really like to see an explicit test that they are configured. I appreciate they are covered by `diffToExpected` so I'm persuadable that its over kill, but it doesn't seem quite right to me.

Reply:
I'm can't say I'm thrilled by these tests myself, but I'm not motivated to refactor them now.  I think what I have done fits the current approach and passes the 'good enough' time.